src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -253,6 +258,10 @@ bool OPT_AppelDuSimplexe(const OptimizationOptions& options,
     long long solveTime = measure.duration_ms();
     optimizationStatistics->addSolveTime(solveTime);
 
+    struct returnSimplexSuccessAndTime returnValues;
+    returnValues.simplexReturnValue = false;
+    returnValues.solveTime = solveTime;
```

Comment:
Don't build here, only in `return` statements `return {.success=true, .solveTime=solveTime};`